auth-web/src/store/modules/user.ts

```diff
@@ -140,7 +140,7 @@ export default class UserModule extends VuexModule {
 
   @Action({ rawError: true })
   public async syncUserProfile () {
-    const userResponse = await UserService.syncUserProfile()
+    const userResponse = await UserService.getUserProfile('@me')
```

Comment:
is this correct?

Reply:
we need to have the syncUserProfile during the initial login, right?
this change might never update the user profile

Reply:
User profile update happens from common components, not from auth-web. The change in common components is to make sure user profile is updated regardless of which app the user logs in.

Reply:
There were lot of calls to syncUserProfile and was causing unwanted update to user profile on backend.

Reply:
oh, you added the post call there. I wasn't aware of that.
That will be good.
I am approving this.